src/phenCryo.ttl

```diff
@@ -41,33 +43,74 @@ sophcr:Calving rdf:type owl:Class ;
 ###  http://sweetontology.net/phenCryo/Englacial
```

Comment:
Should be:
```suggestion
###  http://sweetontology.net/phenCryo/EnglacialProcess
```

Reply:
This only changed the turtle comment to EnglacialProcess? I changed the class name to EnglacialProcess and created Englacial as owl:deprecated "true" + equivalentClass to EnglacialProcess.  Is this what you meant to do @rduerr ?

Reply:
yes - if that's the appropriate SWEET procedure...

Reply:
Yeah. I'm not sure there is an "official" SWEET procedure. Sent @lewismc a message in the ESIP slack channel to begin a discussion. OBO foundry has a procedure (http://www.obofoundry.org/principles/checks/fp_003)
1) Add an owl:deprecated annotation with a boolean value of true to the problematic term
2) Add obsolete to the beginning of the term’s label to prevent duplicating labels
3) Create a new term with a valid IRI to replace this old term
4) Copy the old annotations (label, definition, etc., excluding the owl:deprecated) over to the new term
5) Add a IAO:0100001 (term replaced by) annotation to the old term with a value of the new term’s IRI
We don't have an annotation property like "term replaced by" that would be really useful in this instance. Lewis is suggesting rdfs:seeAlso as a potential replacement.

Reply:
I created a new ticket for this discussion, see https://github.com/ESIPFed/sweet/issues/192. Suggest we move any further discussion of this topic there.